source/features/show-names.tsx

```diff
@@ -61,11 +64,13 @@ async function updateLinks(found: HTMLAnchorElement[]): Promise<void> {
 			continue;
 		}
 
+		const fullNameWithoutEmoji = fullName.replace(emojiRegex, '').trim();
```

Comment:
Do the replacement here so `isUsernameAlreadyFullName` below can also benefit i.e. `johndoe (John Doe 👾)` is now just `John Doe`